ZenPacks/zenoss/ZenPackLib/lib/base/ComponentBase.py

```diff
@@ -231,6 +231,13 @@ def get_containing_relname(self):
         """Return name of containing relationship."""
         for relname, relschema in self._relations:
             if issubclass(relschema.remoteType, ToManyCont):
+                # skip if it's not the real containing rel
+                # for instance an inherited os.filesystems for sub-classed FileSystem
+                rel = getattr(self, relname, None)
```

Comment:
Understood but at the moment this is the only manifestation of an issue and it's a minor one.  I am estimating a fair amount of time needed to understand the scope of impact, what the options are,  and whether any cure would be worse than the disease.

The issue was originally reported by Doug S. and took a few hours to unravel.